linker/Tests/Mono.Linker.Tests.Cases/TypeForwarding/MissingTargetReference.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.TypeForwarding
+{
+	[Define ("IL_ASSEMBLY_AVAILABLE")]
+	[SetupCompileBefore ("TypeForwarderMissingReference.dll", new [] { "Dependencies/TypeForwarderMissingReference.il" })]
+	[SetupLinker ("-p link TypeForwarderMissingReference.dll")]
```

Comment:
I don't like X number of custom attribute to mirror all input arguments. I want the setup to be as simple and readable as possible. Already with few of existing ones it's pain to track down to see what they. Why do you need to translate the arguments, why not just call existing code you have and pass the string[] as it's